recipe/meta.yaml

```diff
@@ -41,6 +46,7 @@ about:
   license_file: LICENSE
   summary: Pytest plugin for measuring coverage
```

Comment:
missing `license_family`

Reply:
The `license_family `is already here: `MIT`